macros/PGbasicmacros.pl

```diff
@@ -2579,11 +2579,11 @@ sub htmlLink {
 
 sub knowlLink {
     # an new syntax for knowlLink that facilitates a local HERE document
-	#   suggested usage   knowlLink(text, [url => ...,   value => ....])
+	#   suggested usage   knowlLink(text, [url => ...,   value => ..., type => ...])
 	my $display_text = shift;
 	my @options = @_;  # so we can check parity
 	my %options = @options;
-	WARN_MESSAGE('usage   knowlLink($display_text, [url => $url,   value => $helpMessage] );'.
+	WARN_MESSAGE('usage   knowlLink($display_text, [url => $url, value => $helpMessage, type => 'help/hint/solution/...'] );'.
```

Comment:
There is an issue with the quotes used here.  It should be:

```suggestion
	WARN_MESSAGE('usage   knowlLink($display_text, [url => $url, value => $helpMessage, type => "help/hint/solution/..."] );'.
```